docs/Drawer.md

```diff
@@ -35,6 +35,7 @@ import { Drawer, DrawerHeader, DrawerSubheader, DrawerBody, DrawerNavGroup, Draw
 | Prop Name                       | Description                                                | Type      | Required | Default  |          
 |---------------------------------|------------------------------------------------------------|-----------|----------|----------|
 | open                            | Controls the open/closed state of the drawer               | `boolean` | yes      |          |
+| variant                         | The variant to use                                         | `'permanent' | 'persistent' | 'temporary'`| no | 
```

Comment:
we should add a blurb/callout explaining the difference between our implementation of 'persistent' vs. what MUI does